extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/GrpcServerRecorder.java

```diff
@@ -256,7 +256,7 @@ public void handle(Void unused) {
             }
         }
 
-        LOGGER.info("Starting new Quarkus gRPC server (using Vert.x transport)...");
+        LOGGER.debug("Starting new Quarkus gRPC server (using Vert.x transport)...");
```

Comment:
Although, this one does tell us which type of gRPC server did we start ... so it's useful ...

Reply:
Why don't you include all information on the same line here: https://github.com/quarkusio/quarkus/pull/46409/files#diff-25de053a1db1dc673c79dfb3b664eeb4bca7c9dc7b6d0ce5ee6649223efee5f9R340 .

Reply:
That one requires "provider" param, which is not available with Vert.x gRPC.

Reply:
Does it? You explicitly test if it is null and have another path for it?

Maybe I misunderstand things but I think I agree with Stuart this message should be debug. And if you want to include more information, it should all be done in the `Started ` message.

Reply:
> but I think I agree with Stuart this message should be debug

Nah, imho, this one should be info, as that'sa the only log / line that tells us what kind of gRPC server we're running - old vs new.

Reply:
Thus why I said this info should be included in the `Started` message. Now I don't know this code but I'm not sure why we couldn't include all the info in one nice log line.

Reply:
And `configuration` is not used -- as we use existing Vert.x http(s) server,
hence misleads the user -- printing out wrong port ...

The initial simple info log is there for a reason ^

Reply:
> Nah, imho, this one should be info, as that'sa the only log / line that tells us what kind of gRPC server we're running - old vs new.

Unless things have changed I thought the consensus was to not include that sort of information in the logs that can be inferred from the application config. If every extension starts logging similar startup would get very noisy.